### PR TITLE
MGMT-8156: Merge translation files from assisted-ui-lib into one file

### PR DESCRIPTION
**Pull request by ammont82**
Related to https://github.com/openshift-assisted/assisted-ui-lib/pull/1399

Related to https://issues.redhat.com/browse/MGMT-8156

Signed-off-by: Montse Ortega <mortegag@redhat.com>

### CODE REVIEW

File: frontend/webpack.config.ts [hunk 1]
```diff
@@ -12,11 +12,12 @@ import ReactRefreshTypeScript from 'react-refresh-typescript'
 import webpack from 'webpack'
 import { BundleAnalyzerPlugin } from 'webpack-bundle-analyzer'
 import { Configuration as DevServerConfiguration } from 'webpack-dev-server'
+import MergeJsonWebpackPlugin from 'merge-jsons-webpack-plugin';
```
Comment (KevinFCormier) [inline frontend/webpack.config.ts:15]: Typo in package name

Comment (ammont82) [reply frontend/webpack.config.ts:15]: Ok!

**Pull request merged**